verilog/analysis/checkers/struct_union_name_style_rule.h

```diff
@@ -51,6 +54,8 @@ class StructUnionNameStyleRule : public verible::SyntaxTreeLintRule {
   static const char kMessageStruct[];
   static const char kMessageUnion[];
 
+  std::vector<std::string> exceptions_;
```

Comment:
I think you can make this a `std::set<>`, and the `absl::StrSplit()` called in `Configure()` will automagically assign it to that set.

That will make finding things in that set simple (and fast, though given that there are not many elements, it probably doesn't matter much; so it would mostly emphasize that we're conceptually mean a set here)